### PR TITLE
Add lastmod to sitemap

### PR DESCRIPTION
**Pull request by melindafekete**
Add the lastmod value to our sitemap using docusaurus presets.

[Preview of the sitemap](https://unleash-docs-git-melinda-sitemap-unleash-team.vercel.app/sitemap.xml)

### CODE REVIEW

File: website/docs/feature-flag-tutorials/flutter/a-b-testing.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ In your Unleash instance, create a new feature flag called `likeOptionExperiment
 
 ![Set Up Variant in Unleash](/img/variant-setup-1.png)
 
-Now that you have created your feature flag, let’s create two new [variants](https://docs.getunleash.io/reference/feature-toggle-variants) “gridTile'' and “imageDetails” respectively. These variants will help you position your “like image” button.
+Now that you have created your feature flag, let’s create two new [variants](https://docs.getunleash.io/reference/feature-toggle-variants) `gridTile` and `imageDetails` respectively. These variants will help you position your **like image** button.
```
Comment (melindafekete) [inline website/docs/feature-flag-tutorials/flutter/a-b-testing.md:25]: Sorry just some unrelated fixes to trigger a new build

File: website/docusaurus.config.ts [hunk 2]
```diff
@@ -295,6 +295,7 @@ const config: Config = {
                     remarkPlugins: [[pluginNpm2Yarn, { sync: true }]],
                     docItemComponent: '@theme/ApiItem',
                     sidebarPath: './sidebars.ts',
+                    showLastUpdateTime: true,
```
Comment (melindafekete) [inline website/docusaurus.config.ts:298]: @alvinometric @nnennandukwe wdyt of having the last updated date shown at the bottom of each page?

Comment (alvinometric) [reply website/docusaurus.config.ts:298]: I'm hesitant, I like seeing it but at the same time I don't want to give the impression that some of our content is outdated

Front-end tech in particular moves so fast that 1-year-old content is often out of date, which isn't the case with our SDKs and our tutorials

**Review by alvinometric — APPROVED** [review]
🎉

File: website/docusaurus.config.ts (hunk 2 above)
Comment (melindafekete) [reply website/docusaurus.config.ts:298]: You're right, maybe in a  few months if/when I've had the chance to make updates to the majority of the content, then we can reconsider
